docs/pages/eas/hosting/reference/responses-and-headers.mdx

```diff
@@ -64,15 +64,50 @@ EAS Hosting will respond with an error page in these cases. The error page will
+| `eas-colo`          | Code of the Cloudflare data center that handled the request. For example, `lhr`                                                                                                                                                            |
+| `eas-ip-continent`  | Two letter continent code of the client.<br /> One of: `AF`, `AN`, `AS`, `EU`, `NA`, `OC`, or `SA`                                                                                                                                         |
+| `eas-ip-country`    | Three letter country code of the client in ISO-3166 Alpha 2 format.<br />For example, `US` or `JP`                                                                                                                                         |
+| `eas-ip-region`     | Region code of the client in ISO-3166-2 format, which has a maximum length of three characters                                                                                                                                             |
+| `eas-ip-city`       | Human-readable city name of the client (optional). For example, `London` or `Chicago`                                                                                                                                                      |
+| `eas-ip-latitude`   | Best guess of the client's latitude (optional)                                                                                                                                                                                             |
+| `eas-ip-longitude`  | Best guess of the client's longitude (optional)                                                                                                                                                                                            |
+| `eas-ip-timezone`   | Timezone of the client. For example, `Europe/London`                                                                                                                                                                                       |
+| `eas-ip-eu`         | Set to `1` when the request likely originated in the jurisdictional area of the European Union                                                                                                                                             |
+
+### Request URL and origin
+
+EAS Hosting routes requests from several hostnames to your deployments. [Aliases](/eas/hosting/deployments-and-aliases) and [Custom domains](/eas/hosting/custom-domain) mean that there may be a difference between the **incoming** URL that clients have used to make a request, and the **target** URL that your API routes receive.
+
+For example, while a client may make a request to an alias URL such as `https://my-app--staging.expo.app/`, the worker deployment that may receive the request will have a URL containing its deployment ID, such as `https://my-app--or1170q9ix.expo.app/`.
```

Comment:
```suggestion
For example, while a client may make a request to an alias URL such as `https://my-app--staging.expo.app/`, the worker deployment that will receive the request will have a URL containing its deployment ID, such as `https://my-app--or1170q9ix.expo.app/`.
```

Are we unsure that the worker will receive the request containing the URL's deployment ID?